.github/workflows/test-suite-desktop-e2e.yml

```
@@ -36,39 +36,88 @@
-          - TEST_GROUP: "@group=wallet"
-            CONTAINERS: "trezor-user-env-unix bitcoin-regtest"
-
     steps:
-        # Electron requires unprivileged user namespaces to function properly. 
-        # Disabling this security rule allows Electron to create sandboxed processes
-        # without requiring elevated privileges, which is essential for running the application.
-        # This is workaround until electron builder solves this issue in future release.
+      - uses: actions/checkout@v4
+      - name: Check previous runs
+        id: check-previous-runs
+        uses: './.github/actions/check-previous-test-runs'
```

actionlint (1.7.4)

missing input "github_token" which is required by action "Check Previous Runs" defined at "./.github/actions/check-previous-test-runs". all required inputs are "github_token", "job_name", "pr_branch", "repo", "workflow_name"

(action)